src/main/java/Topic.java

```diff
@@ -0,0 +1,39 @@
+public class Topic {
+
+    private int id;
```

Comment:
Samce comments like before - 
use only camel case
keep only getter&setters that are in use.